esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -57,6 +57,26 @@ diagnostics:
     scripts: null
 
 
+  E-OBS:
+    description: E-OBS check
+    variables:
+      tas:
+      tasmax:
+      tasmin:
+      pr:
+      psl:
+    additional_datasets:
+      - {dataset: E-OBS, project: OBS, mip: day, tier: 2,
+         type: ground, version: 20.0e_0.1, start_year: 1950, end_year: 2019, end_month: 07}
+      - {dataset: E-OBS, project: OBS, mip: day, tier: 2,
+         type: ground, version: 20.0e_0.25, start_year: 1950, end_year: 2019, end_month: 07}
+      - {dataset: E-OBS, project: OBS, mip: Amon, tier: 2,
+         type: ground, version: 20.0e_0.1, start_year: 1950, end_year: 2019, end_month: 07}
+      - {dataset: E-OBS, project: OBS, mip: Amon, tier: 2,
+         type: ground, version: 20.0e_0.25, start_year: 1950, end_year: 2019, end_month: 07}
```

Comment:
Are the last two entries duplicates of the first two?

Reply:
I wrote them for the Amon mip, not sure if this is needed.

Reply:
Ah, sorry, my bad.
It's correct.